src/BeanstalkClient.php

```diff
@@ -244,4 +247,71 @@ public function ignore(string $tube): Promise {
             }
         });
     }
+
+    public function statsJob(int $id): Promise {
```

Comment:
Currently we follow the original names, but also have verbs for every method name. What do you think about `getJobStats()` / `getTubeStats()` / `getSystemStats()`?

Other than that, everything is fine now. :+1:

Reply:
No problem, if it's your standard way to do it ;)

Reply:
I don't really care, just stumbled over it because of `stats()`, the other two are totally fine. Let's change it and I'll hit the merge button. :+1: